### DISCUSSION TITLE
Is there a way to hide non-user created items in the list?

### DISCUSSION THREAD

**Discussion by andoz1** [johnlindquist/kit#1493] [Q&A]
Hi, 

Is there a way to hide items in the script list?

Basically, I'd like to hide anything and everything except the few scripts I added myself. 

![Screen Shot 2024-06-17 at 10 14 01 - CleanShot - CleanShot](https://github.com/johnlindquist/kit/assets/149333902/ce7b70a0-1c89-4dd8-9e39-d06825f737f8)

**Comment by johnlindquist** [comment]
Not currently... I'll try to get that into the next release though 👍
